### ISSUE TITLE
A11y_AzureDataStudio_CreateAzureKubernetesServiceCluster_ScreenReader:NVDA screen reader focus is not going in logical order.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7658]
"[Check out Accessibility Insights!](https://accessibilityinsights.io) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**#A11y_ADS_OctTestPass;#A11yMAS;#A11yTCS;#SQLAzureDataStudio;#MAS1.3.2;#DesktopApp;#Win32;#NVDA;#JAWS;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
NVDA Version:2019.2.1

**Repro Steps:**

1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button and again click on next button and click on script to Notebook.
8.	 Navigate on the "Create Azure Kubernetes service cluster" screen using tab.
9.  Observed keyboard focus order is not in logical order.

**Actual:**
While NVDA screen reader is on,screen reader focus  order in not in meaning full sequence while navigating on the screen using tab navigation.

**Expected:**
While NVDA screen reader is on,screen reader focus orders should be in  meaning full sequence while navigating on the screen using tab navigation.

**Same issue observed in jaws.**

**User Impact:**
Screen reader only user does not understand the proper logical sequence while navigation.

**Attachment for Reference:**
[7658_A11y_AzureDataStudio_CreateAzureKubernetesServiceCluster_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3742684/7658_A11y_AzureDataStudio_CreateAzureKubernetesServiceCluster_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
@chlafreniere  @corivera , another one.

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;
